examples/source/1.components/amp-script/index.html

```diff
@@ -282,6 +318,43 @@ <h2 id="translated-greeting"></h2>
         greeting.innerHTML = translation + '!';
       </script>
     </div>
+    <!--
+        ## Interacting with AMP components
+        Your script can communicate with an AMP component using a state variable.
+        Bind an attribute in the component to an expression containing a state variable.
+        When your script modifies that state variable, the change will propagate to the component.
+    -->
+    <div id="carousel-sample">
+      <amp-carousel
```

Comment:
Can you simplify this example? We don't need so much markup just to demonstrate the point. A single image with a binding src value would be sufficient.

Reply:
To you and me, `<amp-img>` is an AMP component. But I think that to most people it's just a replacement for `<img>`. And what I wanted to show here was that you can use `<amp-script>` to supercharge an existing AMP component - to give it new features. That people should really try to do this instead of creating components from scratch in JS.

Of course, your example is super clear. In the new documentation, I do something similar - I just show it's possible to use amp-script to modify text in a `<p>`.

Maybe I should include your example too - showing an image changing - for a simple example showing how amp-script can interact with the DOM via data binding. Because I don't show that. And then I could include my example to show how it's possible to modify a component's behavior. Thinking about it more, I should even put the "Surprise me" button on top of the carousel, so we really see the component has gained a new feature.

Reply:
I added the simpler sample showing an image changing. I think this is an improvement - thanks!